.github/workflows/contracts.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           docker run --rm -v "$(pwd)":/code \
             --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
             --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-            cosmwasm/workspace-optimizer:0.12.10
+            cosmwasm/workspace-optimizer:0.16.0
```

Comment:
I have a feeling this will fail for some of the contracts in `/cosmwasm`. I tried updating this [here](https://github.com/osmosis-labs/osmosis/pull/8681/commits/33b9c4c1cf235b98c9249aa4cdd7398956c98b6c) and it failed, trying it locally also failed.

Updating the other contracts (in some fashion) may require a separate PR to make this work.